test/uk/gov/hmcts/pipeline/deprecation/WarningCollectorTest.groovy

```diff
@@ -57,13 +57,13 @@ class WarningCollectorTest extends Specification {
     assertThat(message).isEqualTo(nextDayFormattedDate+" ( tomorrow )")
   }
 
-  def "getMessageByDays() with a week later should return in 6 days"() {
+  def "getMessageByDays() with a week later should return in 7 days"() {
```

Comment:
Ah good! This always seemed wrong to me. The tests were a bit flakey before in that sometimes they'd return 7 and sometimes return 6 - presume that's fixed now?

Reply:
I think so, I changed it to take the time at the start of the day